src/sagas/company.ts

```diff
@@ -23,8 +24,12 @@ function* getCompany(action: GetCompanyAction) {
       yield put(companyActionCreators.getCompanySuccess(response.data.company));
     } else if (!response.ok) {
       yield put(companyActionCreators.getCompanyFailure('company Error'));
+      yield put(
+        errorActionCreators.handleError('getCompanyError', response.data || 'company Error'),
```

Comment:
move getCompanyError to constants (there should be an Object with error names)

const ERROR_KEYS = {
  getCompanyError: 'GET_COMPANTY_ERROR',
  someOtherError: 'ERROR'
}